src/Kaleidoscope.h

```diff
@@ -140,12 +141,18 @@ extern Kaleidoscope_ Kaleidoscope;
 
 /* -- DEPRECATED aliases; remove them when there are no more users. -- */
 
-void event_handler_hook_use(Kaleidoscope_::eventHandlerHook hook)
+void event_handler_hook_use(kaleidoscope::Kaleidoscope_::eventHandlerHook hook)
 __attribute__((deprecated("Use Kaleidoscope.useEventHandlerHook instead")));
-void loop_hook_use(Kaleidoscope_::loopHook hook)
+void loop_hook_use(kaleidoscope::Kaleidoscope_::loopHook hook)
 __attribute__((deprecated("Use Kaleidoscope.useLoopHook instead")));
 
 void __USE_PLUGINS(KaleidoscopePlugin *plugin, ...)
 __attribute__((deprecated("Use Kaleidoscope.use(...) instead")));
 
 #define USE_PLUGINS(...) __USE_PLUGINS(__VA_ARGS__, NULL)
+
+// Use this function macro to register plugins with Kaleidoscope's
+// hooking system. The macro accepts a list of plugin instances that
+// must have been instanciated at global scope.
+//
+#define KALEIDOSCOPE_INIT_PLUGINS(...) _KALEIDOSCOPE_INIT_PLUGINS(__VA_ARGS__)
```

Comment:
This macro is used in the sketch at global scope in global namespace to register plugins. 
Could be renamed to `KALEIDOSCOPE_REGISTER_PLUGINS(...)`. Maybe that's a better name.